README.md

```diff
@@ -164,7 +145,7 @@ pyenv deactivate                 # to exit the virtual environment
 sudo pip install awsebcli        # install the eb cli tools
 ```
 
-Using the Elastic Beanstalk CLI is quite simple but mus tbe done *outside* the virtual environment of the project itself.  It will use the requirements.txt to ensure any requirements are at the right version for the deployed application.
```

Comment:
Do we know if ElasticBeanstalk works without a `requirements.txt`

Reply:
I just checked and it doesn't 😞 
I think we should update the docs here with a line about generating one - https://docs.pipenv.org/advanced.html#generating-a-requirements-txt. We can update Jenkins to do the same.

Reply:
Could we generate a `requirements.txt` file as part of the travis build / provisioning?